docs/content/preview/architecture/transactions/read-committed.md

```diff
@@ -27,24 +27,24 @@ The following two key semantics set apart Read Committed isolation from Repeatab
+In addition to the two key requirements, there is an extra YSQL-specific requirement for read committed isolation level: ensure that external clients don't face `kReadRestart` errors that stem from clock skew which is inherent in distributed databases due to the distribution of data across more than one physical node. PostgreSQL doesn't require defining semantics around read restart errors because it is a single-node database without clock skew. When there is clock skew, the following situation can arise in a distributed database like YugabyteDB:
 
-* A client starts a distributed transaction by connecting to YSQL on a node `N1` in the YugabyteDB cluster and issuing a statement which reads data from multiple shards on different physical YB-TServers in the cluster. For this issued statement, the read point which defines the snapshot of the database at which the data will be read, is picked on a YB-TServer node `M` based on the current time of that YB-TServer. Depending on the scenario, node `M` could be the same as `N1` or not, but that is not relevant to this discussion. Consider `T1` to be the chosen read time.
+* A client starts a distributed transaction by connecting to YSQL on a node `N1` in the YugabyteDB cluster and issues a statement which reads data from multiple shards on different physical YB-TServers in the cluster. For this issued statement, the read point which defines the snapshot of the database at which the data will be read, is picked on a YB-TServer node `M` based on the current time of that YB-TServer. Depending on the scenario, node `M` could be the same as `N1` or not, but that is not relevant to this discussion. Consider `T1` to be the chosen read time.
 * The node `N1` might collect data from many shards on different physical YB-TServers. In this pursuit, it will issue requests to many other nodes to read data.
-* Assuming that node `N1` reads from node `N2`, it could be the case that there exists some data written on node `N2` at time `T2` (> `T1`) but had been written before the read was issued. This can happen because of clock skew where the physical clock on node `N2` might be running slightly ahead of node `M`, and hence the write which was actually done in the past, still has a timestamp higher than `T1`.
+* Assuming that node `N1` reads from node `N2`, it could be the case that there exists some data written on node `N2` with a write timestamp `T2` (> `T1`) but it had been written before the read was issued. This can happen because of clock skew where the physical clock on node `N2` might be running ahead of node `M`, and hence the write which was actually done in the past, still has a write timestamp higher than `T1`.
```

Comment:
Suggestion:

* Assuming that node `N1` reads from node `N2`, it is possible that data had already been written on node `N2` with a write timestamp `T2` (> `T1`) before the read was issued. This could be caused by clock skew if the physical clock on node `N2` ran ahead of node `M`, resulting in the write done in the past still having a write timestamp later than `T1`.

Reply:
nice suggestion! made it much better